sim/common/sod/item_effects/phase_3.go

```diff
@@ -117,5 +119,25 @@ func init() {
 		})
 	})
 
+	// Serpent's Striker
+	itemhelpers.CreateWeaponProcSpell(SerpentsStriker, "Serpent's Striker", 5.0, func(character *core.Character) *core.Spell {
+		procAuras := character.NewEnemyAuraArray(core.SerpentsStrikerFistDebuffAura)
+
+		return character.RegisterSpell(core.SpellConfig{
+			ActionID:    core.ActionID{SpellID: 447894},
+			SpellSchool: core.SpellSchoolNature,
+			DefenseType: core.DefenseTypeMagic,
+			ProcMask:    core.ProcMaskEmpty,
```

Comment:
We should add the `core.SpellFlagPoison` flag to this so that it's registered as a poison